packages/modular-scripts/src/esbuild-scripts/start/index.ts

```diff
@@ -151,6 +156,12 @@ class DevServer {
       define: {
         global: 'window',
       },
+      banner: {
+        js: `window.process = {
```

Comment:
This probably needs a todo, web pack actually shims the process variable / other node.js concepts which is how they're defined. This is probably fine thought.

Reply:
Yeah, that's the minimum viable injection to make react-error-overlay work.
There's a shim for it, yes (it even mocks complex stuff stuff like `hrtime` via the browser's timers) but our main problem with the newest version of reo is that the iframe tries to access `process`, and it's not trivial to inject it there.